.generator/src/generator/templates/model_utils.j2

```
@@ -173,6 +173,18 @@ class OpenApiModel(object):
         """Get the value of an attribute using dot notation: `instance.attr`."""
         return self.__getitem__(attr)
 
+    @overload
+    def __new__(cls, arg: None) -> None:
+        ...
```

Is there a difference between this as `pass`?

---

nope! `...` (or the singleton `Ellipsis`) in python is just a convention used to indicate that it is not the real implementation, whereas `pass` is usually used when the implementation is intentionally blank (like with `try: ... except: pass`)

---

tldr they both do nothing, but conventionally you would use ellipses for overload signature bodies

---

The docs for [overload](https://docs.python.org/3/library/typing.html#typing.overload) also use `...` in their example usage.